packages/freezed/example/lib/main.dart

```diff
@@ -19,7 +19,7 @@ class Union with _$Union {
   factory Union.fromJson(Map<String, Object?> json) => _$UnionFromJson(json);
 }
 
-@freezed
+@Freezed(shouldGenerateMaybeMap: false)
```

Comment:
Shouldn't we add a separate example to demonstrate this use case?
I feel like we are removing an existing example with the default behaviour right ?

Reply:
Also, should we add something to the documentation to describe this new behaviour?